dev/cloud/naming_cenventions.md

```
@@ -0,0 +1,259 @@
+| ------------------------------------------ | --- | 
+| Request has body                           | Yes |     
+| Successful response has body               | Yes |     
+| Safe (Does not change state of the Server) | No  |     
+| Idempontent                                | No  |     
+| Cacheable                                  | No  | 
+A __Post__ Request should be used when a resource is added to the microservice.
+A Post request can also be used when a resource is get but the query would no longer 
+fit in the URL and requires a request body.
+
+##### PUT
+|                                            |     |     |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)